src/arch/arm64/vm-arch.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#define SZ_64K (1UL << 16)
```

Comment:
Is this macro used?

Reply:
If this macro is required, you shall wrap it before using. i.e.,
```c
#ifdef SZ_64K
#define SZ_64K (1UL << 16)
#endif
```

Reply:
This macro is used by other macros in the header file "asm/kvm.h".
I pushed a new revision and wrap `#define SZ_64K ...` as following:
```c
#ifndef SZ_64K
#define SZ_64K (1UL << 16)
#endif
```

You mentioned `#ifdef`, which I think is a typo.